### ISSUE TITLE
Query result podOwnerKind is null

### ISSUE THREAD

**Issue by undera** [otterize/network-mapper#78]
I have a network-mapper instance v0.1.12 and I'm trying to build a correct GraphQL query for its API. 
I'm trying to make query to return intents with corresponding kind field. I have composed the following request:
```graphql
query ServiceIntentsUpToMapperV017 ($namespaces: [String!]) {
	serviceIntents(namespaces: $namespaces) {
		client {
			... NamespacedNameFragment
		}
		intents {
			... NamespacedNameFragment
		}
	}
}
fragment NamespacedNameFragment on OtterizeServiceIdentity {
	name
	namespace
    podOwnerKind { 
      kind
    }
}
```

But whenever I execute it against mapper, the "podOwnerKind" field is always null. I expect to get the valid podOwnerKind for all the cases, be it just orphaned pod, deployment, job etc.

Evyatar Meged has suggested on Slack that it's a real bug.

Corresponding Slack thread: https://otterizecommunity.slack.com/archives/C046SG6PRJM/p1677597181342809

**Comment by amit7itz** [comment]
Hi @undera, thanks for reporting the issue. We just released v0.1.13 which includes a fix

**Comment by undera** [comment]
Wow, that was fast!
Thanks, I already tried and it works as expected.
